attribute/key.go

```diff
@@ -117,6 +117,17 @@ func (k Key) StringSlice(v []string) KeyValue {
 	}
 }
 
+// ByteSlice creates a KeyValue instance with a BYTESLICE Value
+//
+// If creating both a key and value at the same time, use the provided
+// convenience function instead -- ByteSlice(name, value).
+func (k Key) ByteSlice(v []byte) KeyValue {
+	return KeyValue{
+		Key:   k,
+		Value: ByteSliceValue(v),
+	}
+}
```

Comment:
This is analogous to a string, should we also create the `ByteSliceSlice(v [][]byte)`?  Doesn't []String methods suffer from the same problem?